### PR TITLE
Migrate unpkg spa dependency to local path

### PR DESCRIPTION
**Pull request by yathindrakodithuwakku**
### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/index.jsp [hunk 1]
```diff
@@ -33,7 +33,7 @@
 <!DOCTYPE HTML>
 <html>
     <head>
-        <script src="https://unpkg.com/@asgardeo/auth-spa@0.3.3/dist/asgardeo-spa.production.min.js"></script>
+        <script src="console/auth-spa-0.3.3.min.js"></script>
```
Comment (brionmario) [inline apps/console/src/index.jsp:36]: Can't we get the `console` part from `webpack` variables?

File: apps/console/src/public/auth-spa-0.3.3.min.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
...lzIHByb2hpYml0ZWQuIiwiUmVxdWVzdHMgY2FuIG9ubHkgYmUgc2VudCB0byByZXNvdXJjZSBzZXJ2ZXJzIHNwZWNpZmllZCBieSB0aGUgYHJlc291cmNlU2VydmVyVVJMc2AgYXR0cmlidXRlIHdoaWxlIGluaXRpYWxpemluZyB0aGUgU0RLLiBUaGUgc3BlY2lmaWVkIGVuZHBvaW50IGluIHRoaXMgcmVxdWVzdCBjYW5ub3QgYmUgZm91bmQgYW1vbmcgdGhlIGByZXNvdXJjZVNlcnZlclVSTHNgIikpfSkpLGlzQXV0aGVudGljYXRlZDooKT0+aS5pc0F1dGhlbnRpY2F0ZWQoKSxyZWZyZXNoQWNjZXNzVG9rZW46bCxyZXF1ZXN0QWNjZXNzVG9rZW46KHQscixuLGMpPT5lKHZvaWQgMCx2b2lkIDAsdm9pZCAwLChmdW5jdGlvbiooKXtjb25zdCBlPXlpZWxkIHMuZ2V0Q29uZmlnRGF0YSgpO3JldHVybiBuJiZlLmVuYWJsZVBLQ0UmJih5aWVsZCBpLnNldFBLQ0VDb2RlKG4sbnVsbCE9Yz9jOiIiKSksdD9pLnJlcXVlc3RBY2Nlc3NUb2tlbih0LG51bGwhPXI/cjoiIixudWxsIT1jP2M6IiciKS50aGVuKCgoKT0+KG8ucmVmcmVzaEFjY2Vzc1Rva2VuQXV0b21hdGljYWxseSgpLGkuZ2V0QmFzaWNVc2VySW5mbygpKSkpLmNhdGNoKChlPT5Qcm9taXNlLnJlamVjdChlKSkpOlByb21pc2UucmVqZWN0KG5ldyBhKCJTUEEtV09SS0VSX0NPUkUtUkFUMS1ORjAxIiwiTm8gYXV0aG9yaXphdGlvbiBjb2RlLiIsIk5vIGF1dGhvcml6YXRpb24gY29kZSB3YXMgZm91bmQuIikpfSkpLHJlcXVlc3RDdXN0b21HcmFudDp1LHJldm9rZUFjY2Vzc1Rva2VuOigpPT5pLnJldm9rZUFjY2Vzc1Rva2VuKCkudGhlbigoKCk9PihvLmNsZWFyUmVmcmVzaFRva2VuVGltZW91dCgpLFByb21pc2UucmVzb2x2ZSghMCkpKSkuY2F0Y2goKGU9PlByb21pc2UucmVqZWN0KGUpKSksc2V0SHR0cFJlcXVlc3RGaW5pc2hDYW
```
Comment (brionmario) [inline apps/console/src/public/auth-spa-0.3.3.min.js:1]: Shall we create an issue to see if we can create a GitHub CI cronjob to automatically update this once a new release is done in the SPA SDK?

**Review by brionmario — APPROVED** [review]

File: apps/console/src/index.jsp (hunk 1 above)
Comment (yathindrakodithuwakku) [reply apps/console/src/index.jsp]: Fixed

File: apps/console/src/public/auth-spa-0.3.3.min.js (hunk 2 above)
Comment (yathindrakodithuwakku) [reply apps/console/src/public/auth-spa-0.3.3.min.js:1]: This is tracked with https://github.com/wso2-enterprise/asgardeo-product/issues/11666

**Pull request merged**